src/metabase/analytics/stats.clj

```diff
@@ -136,6 +125,16 @@
   []
   (boolean (seq (apply dissoc (public-settings/application-colors) ui-colors))))
 
+(defn fetch-plan-info
```

Comment:
This namespace is specifically for our anonymous stats collection, so if this isn't also being used for that, I think it could be moved closer to the email code